tests/unit/modules/network/sonic/test_sonic_lldp_interfaces.py

```diff
@@ -47,6 +47,25 @@ def tearDown(self):
         self.mock_config_edit_config.stop()
         self.mock_get_interface_naming_mode.stop()
 
+    def validate_config_requests(self):
+        lldp_path = 'data/openconfig-lldp:lldp/interfaces/interface=Ethernet1/config/openconfig-lldp-ext:suppress-tlv-advertisement'
```

Comment:
This will only ever check Ethernet1 since that is hard-coded in. If LLDP is set up for other interfaces, won't they not be validated? I see matches for Ethernet0 and Ethernet2 in the unit test code so those will never be validated, only the Ethernet1 requests.
Could you help me better understand what this function is seeking to validate?

Reply:
This function makes sure that the internal payload is also sorted in addition to the sorting of request paths, 
removed the hard coding of interface Ethernet1 to make sure that the sorting is done for all interface request payloads